models/models.go

```diff
@@ -11,11 +11,25 @@ import (
 
 const (
 	// FormatVersion ...
-	FormatVersion = "14"
+	FormatVersion       = "14"
+	StepListItemWithKey = "with"
 )
 
+// WithModel ...
```

Comment:
nit: these useless comments are no longer required by the linter, you don't need to keep writing them. I also like to clean up existing ones if I touch a file.

Reply:
Cleaned up all the modified files in https://github.com/bitrise-io/bitrise/pull/956/commits/25e96f7785ddff7aa77bdb34498c5711242e2232